src/main/java/forestry/cultivation/gui/widgets/GhostItemStackWidget.java

```diff
@@ -30,25 +30,27 @@ public void draw(int startX, int startY) {
 		if (!slot.getHasStack()) {
 			super.draw(startX, startY);
 		}
-		GL11.glDisable(GL11.GL_LIGHTING);
-		GL11.glDisable(GL11.GL_DEPTH_TEST);
-		GL11.glEnable(GL11.GL_BLEND);
+		GlStateManager.pushMatrix();
```

Comment:
`pushMatrix` and `popMatrix` are only needed when there is a GL transform, but I don't see any in the code below.